kex

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# kex is a utility to quickly exec into a kubernetes pod.
+
+[[ -n $DEBUG ]] && set -x
+
+set -eou pipefail
+IFS=$'\n\t'
+
+SCRIPT_DIR="$(dirname "$( readlink -f "${0}" 2>/dev/null || \
+  python -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "${0}" )")"
+
+if [[ -f "${SCRIPT_DIR}/utils.bash" ]]; then
+  source "${SCRIPT_DIR}/utils.bash"
+else
+  source "${SCRIPT_DIR}/../include/utils.bash"
+fi
+
+KEX="${HOME}/.kube/kex"
+
+usage() {
+  cat <<"EOF"
+USAGE:
+  kex             : exec in to the first listed pod
+  kex -ls,--list     : list the available pods
+  kex <NUM>       : exec into a specified pod
+  kex -h,--help   : show this message
+EOF
+  exit 1
```

Comment:
I'd move this exit code to the validation cases below. Because `kex -h` is valid and should return 0 not 1.